WordPress/Classes/Utility/WPMapFilterReduce.h

```diff
@@ -0,0 +1,37 @@
+#import <Foundation/Foundation.h>
+
+typedef id (^WPMapBlock)(id obj);
+typedef BOOL (^WPFilterBlock)(id obj);
+typedef id (^WPReduceBlock)(id accumulator, id obj);
```

Comment:
Is the Reduce block always gonna be dealing with NSNumber instances?. If so, maybe we should replace the `id` with NSNumber.

(And) if instead of NSNumber we're gonna be dealing with native types, such as *NSInteger*, we may wanna add a TODO so that we wire the Generics feature, once the new SDK is out.